crates/jstzd/tests/octez_node_test.rs

```diff
@@ -0,0 +1,53 @@
+#![cfg(not(target_os = "macos"))]
+// This test is disabled for macos because octez binary is not available
+
+use jstzd::task::{octez_node, Task};
+use rand::Rng;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+#[ignore = "ignore for now until octez binaries in CI are set up properly"]
+#[tokio::test(flavor = "multi_thread")]
+async fn octez_node_test() {
+    let tmp_dir = std::env::temp_dir();
+    let ts = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap()
+        .as_nanos();
+    let data_dir_path = tmp_dir.join(format!("jstzd-octez-node-test-{}", ts));
+    std::fs::create_dir(&data_dir_path).unwrap();
+    let log_path = tmp_dir.join(format!("jstzd-octez-node-test-{}-log.txt", ts));
+    let rpc_endpoint = format!(
+        "localhost:{}",
+        10000 + rand::thread_rng().gen_range(0..10000)
+    );
```

Comment:
There's a tiny chance you might land on a port in use. Could we use TcpListener to grab an os allocated  port instead? 
```
let port = TcpListener::bind("127.0.0.1:0")
        .unwrap()
        .local_addr()
        .unwrap()
        .port();
```